est/jcabi-simpledb.est

```diff
@@ -0,0 +1,17 @@
+date: 28-05-2015
+author: Jason Wong
+method: champions.pert
+scope:
+  1: jcabi-simpledb project scaffolding
+  2: Implementation of simpledb AWS SDK wrapper
+  3: Achieve 80% test coverage
+  4: CI integration
```

Comment:
@super132 @longtimeago we should always estimate "from scratch", imagining how much time would it take to develop everything from a fresh start